mteb/tasks/Classification/prt/HateSpeechPortugueseClassification.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import annotations
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks import AbsTaskClassification
+
+
+class HateSpeechPortugueseClassification(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="HateSpeechPortugueseClassification",
+        description="HateSpeechPortugueseClassification is a dataset of Portuguese tweets categorized with their sentiment (2 classes).",
+        reference="https://aclanthology.org/W19-3510",
+        dataset={
+            "path": "hate_speech_portuguese",
+            "revision": "b0f431acbf8d3865cb7c7b3effb2a9771a618ebc",
+        },
+        type="Classification",
+        category="s2s",
+        eval_splits=["train"],
+        eval_langs=["prt-Latn"],
+        main_score="accuracy",
+        date=("2017-03-08", "2017-03-09"),
+        form=["written"],
+        domains=["Social"],
+        task_subtypes=["Sentiment/Hate speech"],
+        license="Not specified",
```

Comment:
Is this really the case?

Reply:
Yes, unfortunately the license is not specified either in the dataset tab or in the GitHub repo.